docs/spec/resolution.md

```diff
@@ -45,19 +45,19 @@
+            3. Retrieve the operation's [_Core Proof File Recovery Entry_](#core-proof-file-recovery-entry) and [_Chunk File Delta Entry_](#chunk-file-delta-entry) from the pre-processed [_Core Proof File_](#core-proof-file) and [Chunk File](#chunk-file) associated with the operation and proceed to validation of the entries, or, if the [_Core Proof File_](#core-proof-file) and [Chunk File](#chunk-file) have yet to be retrieved and processed (e.g. the resolving node is in a [_Light Node_](#light-node) configuration), perform the following steps:
+                1. Using the [`CAS_PROTOCOL`](#cas-protocol), fetch the [_Core Proof File_](#core-proof-file) and [Chunk File](#chunk-files) using the associated [_Core Proof File URI_](#core-proof-file-uri) and [_Chunk File URI_](#chunk-file-uri). If the file is able to be retrieved, proceed, if the file cannot be found:
                     1. Set the _Next Update Commitment_ value to `null`.
                     2. Use the new _Next Recovery Commitment_ in another run of the [recovery and deactivate operation processing](#recovery-deactivate-operation-processing) loop.
-                2. Validate the [Chunk File](#chunk-file) using the [Chunk File Processing](#chunk-file-processing) procedure. If the [Chunk File](#chunk-file) is valid, proceed, if the file is invalid:
+                2. Validate the [_Core Proof File_](#core-proof-file) and [Chunk File](#chunk-file) using the [Chunk File Processing](#chunk-file-processing) procedure. If the [Chunk File](#chunk-file) is valid, proceed, if the file is invalid:
                     1. Set the _Next Update Commitment_ value to `null`.
                     2. Use the new _Next Recovery Commitment_ in another run of the [recovery and deactivate operation processing](#recovery-deactivate-operation-processing) loop.
             4. Validate the [_Chunk File Delta Entry_](#chunk-file-delta-entry). If the [_Chunk File Delta Entry_](#chunk-file-delta-entry) is valid, proceed, if the entry is invalid:
                 1. Let the state of the DID reflect an _Empty DID State_.
                 2. Use the new _Next Recovery Commitment_ in another run of the [recovery and deactivate operation processing](#recovery-deactivate-operation-processing) loop.
-            5. Generate a hash of the decoded [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [Hashing Process](#hashing-process) and ensure the hash equals the value of the [_Recovery Operation Signed Data Object_](#recovery-signed-data-object) `delta_hash` property. If the values are exactly equal, proceed, if the values are not exactly equal:
+            5. Generate a hash of the canonicalized [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [`HASH_PROTOCOL`](#hash-protocol) and ensure the hash equals the value of the [_Recovery Operation Signed Data Object_](#recovery-signed-data-object) `delta_hash` property. If the values are exactly equal, proceed, if the values are not exactly equal:
```

Comment:
again the question of compression... compressed and uncompressed values do not hash to the same identifier.

Reply:
Took care of this with added processing step.